apple/internal/testing/apple_test_rule_support.bzl

```diff
@@ -135,6 +138,10 @@ def _apple_test_rule_impl(ctx, test_type):
 
     test_bundle = test_bundle_target[AppleTestInfo].test_bundle
 
+    test_filter = ctx.attr.test_filter
+    if len(test_filter) == 0:
```

Comment:
I don't think you need this `if`, empty string should be falsy in the function you pass this to, so it should handle it there. Maybe the default should be `None` as well too?